fleet/main.tf

```diff
@@ -196,6 +197,11 @@ resource "aws_autoscaling_group" "spot_fleet" {
       value               = "${var.bootstrap_version}"
       propagate_at_launch = true
     },
+    {
+      key                 = "snapshot_filename"
+      value               = "${var.snapshot_filename}"
```

Comment:
we might hit the same issue where tag of ASG is empty

Reply:
good point, workaround in e8706f4

Reply:
see also https://github.com/aeternity/infrastructure/pull/410/commits/82a0346e6def1aae293e8cfe29633db3f89fb975